src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -74,10 +74,11 @@ first "effective" echo and the center of the last "effective" echo.
 
 #### RF & Contrast
 
-| Field name                  | Definition                                                                                                            |
-| :-------------------------- | :-------------------------------------------------------------------------------------------------------------------- |
-| FlipAngle                   | RECOMMENDED. Flip angle for the acquisition, specified in degrees. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`. |
-| MultibandAccelerationFactor | RECOMMENDED. The multiband factor, for multiband acquisitions.                                                        |
+| Field name                  | Definition                                                                                                                                                                                                                                                                                                                                       |
+| :---------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FlipAngle                   | RECOMMENDED. Flip angle for the acquisition, specified in degrees. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`.                                                                                                                                                                                                                            |
+| MultibandAccelerationFactor | RECOMMENDED. The multiband factor, for multiband acquisitions.                                                                                                                                                                                                                                                                                   |
+| Negative Contrast           | OPTIONAL. Whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol. Values MUST be one of True, False.                  |
```

Comment:
I would make it explicit that this is a `boolean` and values need to be `true` or `false` (See https://www.w3schools.com/js/js_json_datatypes.asp). Is there a DICOM field corresponding to this field?

Reply:
done, and no, no DICOM analogue.